packages/k6/entry.sh

```diff
@@ -36,24 +36,46 @@ fi
 if [ ! -z ${PR_NUMBER+x} ]; then
   echo "pr was set, sending pass/fail data to pr";
   TEST_OUTPUT=$(tail -n 150 ~/testResults.txt)
+  TEST_OUTPUT_ASSERTIONS=$(cat ~/output.log | grep "Failed " | sort | uniq)
```

Comment:
Do we only want to get the failed assertions? Might be useful to look at what assertions succeeded, as long as it's clearly called out if any of them failed?

Reply:
The number of unique assertions applied during the test is in `TEST_OUTPUT_ASSERTIONS_COUNT`. This should be enough to verify the output is tested.

Reply:
Right, but wouldn't it be useful to also see what assertions were made, like we output the k6 test result regardless if it failed or not? Or is the output here so big that it's no longer useful to include?

I don't have too strong opinions on this, but I do feel like it's more useful to provide some output regardless if it fails or not.

Reply:
I've added the list of assertions, this would be helpful. The full log of successful assertions is too long to include.

Reply:
Looks great!